### PR TITLE
nautilus: qa/tests: added client-upgrade-nautilus-pacific tests

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by yuriw** [comment]
http://pulpito.ceph.com/teuthology-2021-03-03_17:01:40-upgrade-clients:client-upgrade-nautilus-pacific-pacific-distro-basic-smithi/

**Comment by smithfarm** [comment]
jenkins test make check

**Comment by yuriw** [comment]
https://pulpito.ceph.com/teuthology-2021-03-24_17:04:50-upgrade-clients:client-upgrade-nautilus-pacific-pacific-distro-basic-smithi/

**Review by jdurgin — APPROVED** [review]
structure looks good, let's see how it runs

**Comment by yuriw** [comment]
I killed the run, it was stuck at and timing out :

2021-03-24T17:26:48.280 DEBUG:teuthology.orchestra.remote:smithi119:/home/ubuntu/cephtest/workunits.list.client.1 is 4KB
2021-03-24T17:26:48.292 INFO:tasks.workunit:Running workunits matching rbd/notify_slave.sh on client.1...
2021-03-24T17:26:48.292 INFO:tasks.workunit:Running workunit rbd/notify_slave.sh...
2021-03-24T17:26:48.292 DEBUG:teuthology.orchestra.run.smithi119:workunit test rbd/notify_slave.sh> mkdir -p -- /home/ubuntu/cephtest/mnt.1/client.1/tmp && cd -- /home/ubuntu/cephtest/mnt.1/client.1/tmp && CEPH_CLI_TEST_DUP_COMMAND=1 CEPH_REF=octopus TESTDIR="/home/ubuntu/cephtest" CEPH_ARGS="--cluster ceph" CEPH_ID="1" PATH=$PATH:/usr/sbin CEPH_BASE=/home/ubuntu/cephtest/clone.client.1 CEPH_ROOT=/home/ubuntu/cephtest/clone.client.1 RBD_FEATURES=61 adjust-ulimits ceph-coverage /home/ubuntu/cephtest/archive/coverage timeout 3h /home/ubuntu/cephtest/clone.client.1/qa/workunits/rbd/notify_slave.sh
2021-03-24T17:26:48.305 INFO:tasks.workunit.client.1.smithi119.stderr:+ dirname /home/ubuntu/cephtest/clone.client.1/qa/workunits/rbd/notify_slave.sh
2021-03-24T17:26:48.306 INFO:tasks.workunit.client.1.smithi119.stderr:+ relpath=/home/ubuntu/cephtest/clone.client.1/qa/workunits/rbd/../../../src/test/librbd
2021-03-24T17:26:48.306 INFO:tasks.workunit.client.1.smithi119.stderr:+ python3 /home/ubuntu/cephtest/clone.client.1/qa/workunits/rbd/../../../src/test/librbd/test_notify.py slave

File: qa/suites/upgrade-clients/client-upgrade-nautilus-pacific/nautilus-client-x/rbd/1-install/nautilus-client-x.yaml
Comment (dillaman) [inline qa/suites/upgrade-clients/client-upgrade-nautilus-pacific/nautilus-client-x/rbd/1-install/nautilus-client-x.yaml:9]:
```suggestion
- print: "**** done install.upgrade to -x on client.1"
```

**Review by dillaman — COMMENTED** [review]
It looks like the Python scripts failed to connect to the cluster for some reason since otherwise you should see "starting master/slave" log out lines after they connect.

**Comment by yuriw** [comment]
@ideepika we want to merge this after your investigation/fixing is done

**Comment by smithfarm** [comment]
@yuriw The second commit in this PR is not signed. Did you forget `--signoff`?

**Comment by ideepika** [comment]
> @ideepika we want to merge this after your investigation/fixing is done

yes @yuriw working on it

**Comment by smithfarm** [comment]
> @yuriw The second commit in this PR is not signed. Did you forget `--signoff`?

@yuriw Ping

**Review by ideepika — APPROVED** [review]
working on fixing failures, but meanwhile do not want to keep things on hold, tests looks good

**Comment by yuriw** [comment]
jenkins test api

**Comment by yuriw** [comment]
test this please

**Comment by yuriw** [comment]
test this please

**Comment by yuriw** [comment]
test this please

**Comment by yuriw** [comment]
test make check

**Comment by yuriw** [comment]
jenkins test make check

**Comment by yuriw** [comment]
jenkins test make check

**Pull request merged**
